index.html

```diff
@@ -1,82 +1,327 @@
-          src="./Image/logo-david-nuevo-2021.png" alt="logo"> </a>
-      
-      <nav>
-        <label for="menu-toggle">MY FIRST WEB PAGE</label>
-        <input id="menu-toggle" type="checkbox" />
-        <label class='menu-button-container' for="menu-toggle">
-          <span class='menu-button'></span>
-        </label>
-        <ul class="menu">
-          <li>Portfolio</li>
-          <li>About</li>
-          <li>Contact</li>
-        </ul>
-      </nav>
+
+    <header>
+        <nav class="navbar">
+            <a href="https://www.linkedin.com/in/david-verde-3349b114b/e"> <img id="mylogo"
+                    src="./image/logo-david-nuevo-2021.png" alt="logo"> </a>
+
+            <ul class="nav-list">
+                <li class="list-item">
+                    <a href="#"> Portfolio </a>
+                </li>
+                <li class="list-item">
+                    <a href="#"> About </a>
+                </li>
+                <li class="list-item">
+                    <a href="#"> Contact </a>
+                </li>
+            </ul>
+            <div class="menu">
+                <div class="menu-line"></div>
+                <div class="menu-line"></div>
+                <div class="menu-line"></div>
+            </div>
+        </nav>
+    </header>
+
+    <section class="headlines">
+        <div class="container">
+            <div class="headline-text">
+
+                <h1 class="primary-text">I’m David Verde<br> Glad to see you!</h1>
+                <p>Using a series of utilities, you can create this jumbotron, just like the one in previous versions of
+                    Bootstrap. Check out the examples below for how you can remix and restyle it to your liking.</p>
+                <a href="#" class="btn-connect">
+                    <h6>LET´S CONNECT</h6>
+                </a>
+                <ul class="small_text">
+
+                    <li>
+                        <a href="https://twitter.com/UnyieldingOne">
+                            <img class="size" src="./image/twitter.svg" alt="" /></a>
+                    </li>
+                    <li>
+                        <a href="https://www.linkedin.com/in/david-verde-3349b114b/e">
+                            <img class="size" src="./image/linkedin.svg" alt="" /></a>
+                    </li>
+                    <li>
+                        <a href="https://medium.com/@david.verde.alvarez">
+                            <img class="size" src="./image/medium.svg" alt="" /></a>
+                    </li>
+                    <li>
+                        <a href="https://github.com/Unyielding1">
+                            <img class="size" src="./image/github.svg" alt="" /></a>
+                    </li>
+                    <li>
+                        <a href="https://angel.co/profile/edit/overview">
+                            <img class="size" src="./image/angellist.svg" alt="" /></a>
```

Comment:
- [ ] Your logo image is not displayed, because your using the wrong path for your image, kindly fix this, by changing the path to ``` "../Image/angellist.svg"```